src/components/Chips/__tests__/chips-snapshot-tests.jest.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Chips from "../Chips";
+import renderer from "react-test-renderer";
+import { Calendar } from "../../Icon/Icons";
+
+describe("Chips renders correctly", () => {
+  it("renders correctly with empty props", () => {
+    const tree = renderer.create(<Chips />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("renders correctly with color", () => {
+    const tree = renderer.create(<Chips color={Chips.colors.NEGATIVE} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  
+  it("renders correctly disabled chip", () => {
+    const tree = renderer.create(<Chips disabled={true} />).toJSON();
```

Comment:
please change the syntax when setting a true boolean value to a prop to short hand syntax like this: `<Chips disabled} />`